Example/openxcframework_UITests/openxcframework_UITests.swift

```diff
@@ -19,13 +19,13 @@ class openxcframework_UITests: XCTestCase {
         // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
     }
 
-    func testserchBleButton() {
+    func testSerchBleButton() {
```

Comment:
You updated the camel case, but the spelling still needs to be corrected - testSearchBleButton. You forgot the "a"